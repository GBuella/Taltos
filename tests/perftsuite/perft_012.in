setboard r3k2r/8/8/8/8/8/8/R3K2R w KQkq - 0 1
echo r3k2r/8/8/8/8/8/8/R3K2R w KQkq - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
