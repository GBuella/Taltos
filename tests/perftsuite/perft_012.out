r3k2r/8/8/8/8/8/8/R3K2R w KQkq - 0 1
26
568
13744
314346
7594526
179862938
